src/folders.js

```diff
@@ -7,7 +7,6 @@ import * as debug from './debug'
 const UI = {
   icons: require('./iconBase'),
   ns: require('./ns'),
-  pad: require('./'),
```

Comment:
Is this change necessary in this PR?

Reply:
I removed 3 unnecessary dependencies at the same time.  Can we let that go through in the same PR?